src/base/utils/misc.cpp

```diff
@@ -588,24 +589,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     // QProcess::startDetached() explorer.exe still shows the default path. In
     // this case QProcess::startDetached() probably puts its own quotes around ours.
 
-    STARTUPINFO startupInfo;
-    ::ZeroMemory(&startupInfo, sizeof(startupInfo));
-    startupInfo.cb = sizeof(startupInfo);
-
-    PROCESS_INFORMATION processInfo;
-    ::ZeroMemory(&processInfo, sizeof(processInfo));
-
-    QString cmd = QString("explorer.exe /select,\"%1\"").arg(Utils::Fs::toNativePath(absolutePath));
-    LPWSTR lpCmd = new WCHAR[cmd.size() + 1];
-    cmd.toWCharArray(lpCmd);
-    lpCmd[cmd.size()] = 0;
-
-    bool ret = ::CreateProcessW(NULL, lpCmd, NULL, NULL, FALSE, 0, NULL, NULL, &startupInfo, &processInfo);
-    delete [] lpCmd;
-
-    if (ret) {
-        ::CloseHandle(processInfo.hProcess);
-        ::CloseHandle(processInfo.hThread);
+    CoInitializeEx(NULL, COINIT_MULTITHREADED);
+    ITEMIDLIST *pidl = ILCreateFromPathW(reinterpret_cast<PCTSTR>(Utils::Fs::toNativePath(path).utf16()));
+    if (pidl) {
+        SHOpenFolderAndSelectItems(pidl, 0, nullptr, 0);
+        ILFree(pidl);
     }
```

Comment:
1. Call CoUnitialize at the end if the return of CoInitializeEx is S_OK or S_FALSE. (check remarks of [CoInitializeEx](https://msdn.microsoft.com/en-us/library/windows/desktop/ms695279(v=vs.85).aspx))
2. I find it better to prefix WINAPI calls with `::`. It makes it better to understand that these are WINAPI. (I know we don't have a rule about this)
3. You have mixed usage of `nullptr` and `NULL`. Since this is WINAPI should we always use `NULL`. @glassez thoughts?
4. I am viewing this on mobile, but I think the code comments above the diff snippet aren't relevant anymore. I talk about the code lines that don't automatically show in the diff.